src/CONST.ts

```diff
@@ -5315,6 +5315,11 @@ const CONST = {
             DONE: 'done',
             PAID: 'paid',
         },
+        CHAT_STATUS: {
+            UNREAD: 'unread',
+            PINNED: 'pinned',
+            DRAFT: 'draft',
```

Comment:
Let's remove draft for now. We don't currently store drafts on the server so there's no way for us to search for them server side.